docs/build-transaction-construction.md

```diff
@@ -0,0 +1,208 @@
+---
+id: build-transaction-construction
+title: Transaction Construction and Signing
+sidebar_label: Transaction Construction
+---
+
+This page will discuss the transaction format in Polkadot and how to create, sign, and broadcast
+transactions. Like the other pages in this guide, this page demonstrates some of the available
+tools. Always refer to each tool's documentation when integrating.
+
+## Transaction Format
+
+Polkadot has some basic transaction information that is common to all transactions.
+
+- Address: The SS58-encoded address of the sending account.
+- Block Hash: The hash of the [checkpoint](build-protocol-info#transaction-mortality) block.
+- Block Number: The number of the checkpoint block.
+- Genesis Hash: The genesis hash of the chain.
+- Metadata: The SCALE-encoded metadata for the runtime when submitted.
+- Nonce: The nonce for this transaction.*
+- Spec Version: The current spec version for the runtime.
+- Tip: Optional, the [tip](build-protocol-info#fees) to increase transaction priority.
+- Validity Period: Optional, the number of blocks after the checkpoint for which a transaction is
+  valid. The default is 300 (5 minutes). If zero, the transaction is
+  [immortal](build-protocol-info#transaction-mortality).
+
+*The nonce queried from the System module does not account for pending transactions. You must track
+and increment the nonce manually if you want to submit multiple valid transactions at the same time.
+
+Each transaction will have its own (or no) parameters to add. For example, the `transferKeepAlive`
+function from the Balances pallet will take:
+
+- `dest`: Destination address
+- `#[compact] value`: Number of tokens (compact encoding)
+
+Once you have all the necessary information, you will need to:
+
+1. Construct an unsigned transaction.
+1. Create a signing payload.
+1. Sign the payload.
+1. Serialize the signed payload into a transaction.
+1. Submit the serialized transaction.
+
+Parity provides the following tools to help perform these steps.
+
+## Polkadot JS Tools
```

Comment:
A common question I've gotten is "how do I do these things without JS?" Maybe should address that.